tests/run.sh

```diff
@@ -76,8 +76,13 @@ show_usage() {
   echo "  -s       actual test script. default: 'yarn test:integration'"
 }
 
+# Find the latest released firmware version
+RELEASED_FIRMWARE=$(curl https://raw.githubusercontent.com/trezor/webwallet-data/master/firmware/2/releases.json | tac | tac | grep -m1 -o -P '(?<="version\": \[).*(?=\])' | sed 's/, /./g')
```

Comment:
won't this still fail if we runt the script in the ci ?

Reply:
it wokrs in github, so I would say that as long as configuration of the base image is ok, ti should pass